app/helpers/thumbnail_helper.rb

```diff
@@ -71,7 +71,8 @@ def image_caption_html(image_id, obs_data, link_type)
   end
 
   def image_observation_data(html, obs_data, link_type)
-    if link_type == :naming || obs_data[:obs].vote_cache <= 0
+    if link_type == :naming ||
+       (obs_data[:obs].vote_cache.present? && obs_data[:obs].vote_cache <= 0)
       html << caption_propose_naming_link(obs_data)
```

Comment:
It may be an artifact of the old improperly stripped db snapshot, but my local install crashed without this.
obs.vote_cache was nil.